identityClient/identityClient.go

```diff
@@ -26,11 +27,13 @@ const (
 	roleMapperResourceName        = "role_mapping"
 	realmLoginPathPrefix          = "/auth/realms"
 	realmLoginPathPostfix         = "/protocol/openid-connect/token"
+	realmLoginAuthPathPostfix     = "/protocol/openid-connect/auth"
 	applicationMapperResourceName = "mapper"
 )
 
 var (
 	internalIdentityServiceBasePath = fmt.Sprintf("/internal%s", identityServiceBasePath)
+	encoder = schema.NewEncoder()
```

Comment:
Can we get a comment and intentional revealing name like `httpFormSchemaEncoder`?